### DISCUSSION TITLE
Importing an external library from Malt

### DISCUSSION THREAD

**Discussion by panthavma** [bnpr/Malt#298] [Q&A] [answered]
Hello! I would like to import an external library obtained through pip from my custom pipeline, but it seems like it can't find it.
That might be blender specific, but would you know how to do it ?

```py
import openmesh as om
# ModuleNotFoundError: No module named 'openmesh'


class CustomPipeline(Pipeline):
# [...]
```

**Comment by pragma37** [comment, marked as answer]
Yes, this is a Blender thing. Blender ships with its own local python installation `<Blender Folder>/<Version Folder>/python/bin/`.

You have to install your library through that python installation `<Blender Folder>/<Version Folder>/python/bin/python -m pip ...`

If you're using Linux you will have to run `-m ensurepip` first.

**Reply by TrevorBlythe** [reply]
Im just trying to enable the addon, and it keeps saying missing module despite installing it systemwide, and on the blender python.

**Reply by panthavma** [reply]
oops I forgot to mark it as resolved on my end, this did work for me on my various linux machines
